### ISSUE TITLE
Form control placeholder colour has incorrect opacity

### ISSUE THREAD

**Issue by lonix1** [twbs/bootstrap#40499]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

A form control's placeholder colour is defined like this:
```css
--bs-secondary-color: rgba(33, 37, 41, 0.75);

.form-control::placeholder {
  color: var(--bs-secondary-color);
  opacity: 1;
}
```
Note how it resets opacity to `1`.

Surely that's a mistake? The placeholder text is expected to be lighter than normal text.

### Reduced test cases

```html
<input type="text" class="form-control" placeholder="hello world!">
```

### What operating system(s) are you seeing the problem on?

Windows, macOS, Android, iOS, Linux

### What browser(s) are you seeing the problem on?

Chrome, Firefox, Microsoft Edge

### What version of Bootstrap are you using?

5.3.3

**Comment by julien-deramond** [comment]
Hello @lonix1 
Thanks for reporting an issue.

I've tried to drop the `opacity: 1` to check the rendering:
![Screenshot 2024-05-28 at 09 10 22](https://github.com/twbs/bootstrap/assets/17381666/db41baab-21e6-4804-80fa-d66677a1a331)
The issue with this lighter gray is that it's not accessible. The contrast between the foreground and background colors is not sufficient.

With the current color, it's AA and not even AAA for small texts:
![Screenshot 2024-05-28 at 09 10 34](https://github.com/twbs/bootstrap/assets/17381666/2e7e8b9a-cbae-4ffd-a1f6-001b082baebd)

So, in terms of pure rendering, I tend to believe that it's the right color to use.

Technically, however, I agree that using a `rgba` value to define the `color` is maybe not the right choice since it becomes semi-transparent and would cause issues, for instance if the text was on top of a colored/image surface. That's maybe something to better tackle in the future by using the equivalent solid color with `rgb` instead of `rgba(..., .75)`.

**Comment by julien-deramond** [comment]
> I assume you are referring to people with colour blindness/contrast problems

Yes, we are trying to define our color palette and color system so that we can respect the [Web Content Accessibility Guidelines (WCAG) with an AAA level whenever possible, or AA minimum](https://www.w3.org/WAI/WCAG2AA-Conformance).
FYI, personally, I use the [Colour Contrast Analyzer](https://github.com/ThePacielloGroup/CCAe) to compare the foreground and background colors.
